docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -158,6 +161,101 @@ The key you provide should look like:
 -----END PRIVATE KEY-----
 ```
 
+## Set up SAML with Azure AD
+
+Grafana supports user authentication through Azure AD, which is useful when you want users to access Grafana using single sign-on. This guide will guide you through the steps of configuring SAML authentication in Grafana with [Azure AD](https://azure.microsoft.com/en-us/services/active-directory/).
+
+**Requirements**
+
+- You need to be an admin in your Azure AD organization to access Azure AD and create SAML integration.
+- You need permissions to edit Grafana config file and restart Grafana server.
+
+**Before you begin:**
+
+- Ensure you have permission to administer SAML authentication. For more information about roles and permissions in Grafana.
+  - [Roles and permissions]({{< relref "../../../../administration/roles-and-permissions" >}}).
+- Learn the limitations of Azure AD SAML integration.
+  - [Azure AD SAML limitations](https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#groups-overage-claim)
+- Configure SAML integration with Azure AD, create an app integration inside the Azure AD organization first.
+  - [Add app integration in Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-configure)
+- If you have users that belong to more than 150 groups, you need to configure a registered application to provide an Azure Graph API to retrieve the groups.
+  - [Setup Azure AD Graph API applications]({{< relref "#set-up-saml-with-azure-ad" >}})
+
+**Generate self-signed certificates**
+Azure AD requires a certificate to sign the SAML requests. You can generate a self-signed certificate using the following command:
+
+```sh
+$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
+```
+
+This will generate a `key.pem` and `cert.pem` file that you can use for the `private_key` and `certificate` configuration options.
+
+### Add Microsoft Entra SAML Toolkit from the gallery
+
+> Taken from https://learn.microsoft.com/en-us/entra/identity/saas-apps/saml-toolkit-tutorial#add-microsoft-entra-saml-toolkit-from-the-gallery
```

Comment:
Not much as copy/pasted, but I wanted to add a reference to the original docs. Just in case our own documentation gets outdated, there's a reference to the original documentation.

Should I rewrite it?